### ISSUE TITLE
webpack hot-reload sometimes crashes to error

### ISSUE THREAD

**Issue by Havunen** [infernojs/ts-plugin-inferno#7]
```
<e> [webpack-dev-middleware] HookWebpackError: Cannot read properties of undefined (reading 'kind')
<e>     at makeWebpackError (DIR.TestAutomation\viewer\node_modules\webpack\lib\HookWebpackError.js:48:9)
<e>     at DIR.TestAutomation\viewer\node_modules\webpack\lib\Compilation.js:3068:12
<e>     at eval (eval at create (DIR.TestAutomation\viewer\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:43:1)
<e>     at fn (DIR.TestAutomation\viewer\node_modules\webpack\lib\Compilation.js:481:17)
<e>     at _next0 (eval at create (DIR.TestAutomation\viewer\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:41:1)
<e>     at eval (eval at create (DIR.TestAutomation\viewer\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:54:1)
<e>     at Hook.eval [as callAsync] (eval at create (DIR.TestAutomation\viewer\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
<e>     at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (DIR.TestAutomation\viewer\node_modules\tapable\lib\Hook.js:18:14)
<e>     at fn (DIR.TestAutomation\viewer\node_modules\webpack\lib\Compilation.js:487:45)
<e>     at Hook.eval [as callAsync] (eval at create (DIR.TestAutomation\viewer\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:50:1)
<e> -- inner error --
<e> TypeError: Cannot read properties of undefined (reading 'kind')
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:50:26)
<e>     at getValue (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\utils\getValue.js:9:16)
<e>     at getVNodeProps (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:453:62)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:162:26)
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:54:28)
<e>     at getVNodeChildren (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:485:29)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:163:35)
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:54:28)
<e>     at getVNodeChildren (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:485:29)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:163:35)
<e> caused by plugins in Compilation.hooks.processAssets
<e> TypeError: Cannot read properties of undefined (reading 'kind')
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:50:26)
<e>     at getValue (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\utils\getValue.js:9:16)
<e>     at getVNodeProps (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:453:62)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:162:26)
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:54:28)
<e>     at getVNodeChildren (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:485:29)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:163:35)
<e>     at visitor (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:54:28)
<e>     at getVNodeChildren (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:485:29)
<e>     at createVNode (DIR.TestAutomation\viewer\node_modules\ts-plugin-inferno\dist\index.js:163:35)
```

**Comment by Havunen** [comment]
This seems to be caused by empty ref statement

**Comment by Havunen** [comment]
Fixed in 6.1.0
